### PR TITLE
support routine load isolation_level read_committed

### LINKED ISSUE #6138: routine load  kafka property 未生效
**Describe the bug**
routine load  kafka property  "property.isolation.level"="read_committed" 无效
**To Reproduce**

**Expected behavior**
希望只读kafak中commit的数据

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by WindyGao**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #6138  https://github.com/apache/incubator-doris/issues/3682  https://github.com/apache/incubator-doris/issues/5485) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by WindyGao** [comment]
**1、produce msg without commitTransaction() => courseId (20,21,22) 3 messages**
image


**1.1、routineLoad with "read_uncommitted"**

- table:
image

**1.2、routineLoad with "read_committed"**

- table: 
image

**Comment by WindyGao** [comment]
**1、produce msg with commitTransaction() => courseId (20,21,22,23,24) 5 messages**
image



**1.1、routineLoad with "read_uncommitted"**

- table:
image


**1.2、routineLoad with "read_committed"**

- table: 
image

**Comment by WindyGao** [comment]
after some research, the rdKafka client cames to the following conclusions

Idempotent Producer - supported in librdkafka v1.0.0 ✅
Producer transactions - supported in v1.4.0 ✅
Consumer transactions - supported in v1.2.0 ✅

File: thirdparty/vars.sh [hunk 1]
```diff
@@ -202,11 +202,17 @@ ROCKSDB_NAME=rocksdb-5.14.2.tar.gz
 ROCKSDB_SOURCE=rocksdb-5.14.2
 ROCKSDB_MD5SUM="b72720ea3b1e9ca9e4ed0febfef65b14"
 
-# librdkafka
-LIBRDKAFKA_DOWNLOAD="https://github.com/edenhill/librdkafka/archive/v0.11.6-RC5.tar.gz"
-LIBRDKAFKA_NAME=librdkafka-0.11.6-RC5.tar.gz
-LIBRDKAFKA_SOURCE=librdkafka-0.11.6-RC5
-LIBRDKAFKA_MD5SUM="2e4ecef2df277e55a0144eb6d185e18a"
+# librdkafka-0.11.6
```
Comment (morningman) [inline thirdparty/vars.sh:205]: It's ok to remove the old code

Comment (WindyGao) [reply thirdparty/vars.sh]: done~

Comment (WindyGao) [reply thirdparty/vars.sh]: TBR
@morningman

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request merged**